speech/transcribeStreaming.v2.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: Transcribe a local file using streaming recognition
+//   description: Streaming transcription of a local file using Speech-to-Text v2
+
+async function main(recognizerName, filename = 'resources/brooklyn.flac') {
```

Comment:
The [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments) recommends minimizing the amount of arguments used.

Can we move `filename` into its own `const` line instead? e.g:
```
// TODO(developer): replace this file path with your own
const filename = 'resources/brooklyn.flac
```

Reply:
We can minimize the number of arguments, but I choose not to. I'm hewing to the "Copy-paste-runnable" goal in the [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/#introduction).

Reply:
Sorry, I misunderstood your comment. Done.